lua/mapvote/server/modules/rtv.lua

```diff
@@ -62,11 +62,13 @@ end
 function RTV.ShouldChange()
     if MapVote.state.isInProgress then return end
     local conf = MapVote.GetConfig()
-    local plyCount = player.GetCount()
-    if plyCount < conf.RTVPlayerCount then return end
     local totalVotes = RTV.GetVoteCount()
     local totalPlayers = RTV.GetPlayerCount()
+
+    if totalPlayers < conf.RTVPlayerCount then return end
+
     if totalPlayers == 0 then return end
```

Comment:
You can remove this too 👍 
```suggestion
```

Reply:
I kept it there in caase that value is ever set to 0 for whatever reason.  dont think theres any cause we would want RTVs to be allowed with 0 voters